automatic/tor-browser/update.ps1

```diff
@@ -23,14 +23,14 @@ function global:au_GetLatest {
   $download_page    = Invoke-WebRequest -Uri $releases -UseBasicParsing
     
   $allExes          = $download_page.Links | Where-Object href -match "\.exe$" | Select-Object -expand href
-  $url32            = $allExes | Where-Object { $_ -match "torbrowser-install-\d.*_All.exe$" } | Select-Object -First 1
-  $url64            = $allExes | Where-Object { $_ -match "torbrowser-install-win64.*_All.exe$" } | Select-Object -First 1
-  $version          = $url64 -split '\/' | Select-Object -last 1 -skip 1
+  $url32            = $releases + ($allExes | Where-Object { $_ -match "tor-browser-windows-x86_64-portable-\d.*.exe$" } | Select-Object -First 1)
+  $url64            = $releases + ($allExes | Where-Object { $_ -match "tor-browser-windows-x86_64-portable-\d.*.exe$" } | Select-Object -First 1)
```

Comment:
Don't insert `$releases` here.

We need to extract the last three parts of the matched URL: `/dist/torbrowser/13.0.6/tor-browser-windows-x86_64-portable-13.0.6.exe` and store it into url32 and url64